zone/bot.cpp

```diff
@@ -8255,7 +8248,7 @@ bool Bot::GetNeedsHateRedux(Mob *tar) {
 	//		return false;
 	//	}
 	//}
-	//else if (tar->IsBot()) {
+	//else if (tar->is_bot()) {
```

Comment:
Why was this changed?

Reply:
same as above, renamed due to no context since it's a comment, I ended up just removing the commend cause it's pointless